src/schema/slack/functions/canvas_update_content.ts

```diff
@@ -0,0 +1,56 @@
+/** This file was autogenerated. Follow the steps in src/schema/slack/functions/_scripts/README.md to rebuild **/
+import { DefineFunction } from "../../../functions/mod.ts";
+import SchemaTypes from "../../schema_types.ts";
+import { SlackPrimitiveTypes } from "../../slack/types/mod.ts";
+import SlackTypes from "../schema_types.ts";
+
+export default DefineFunction({
+  callback_id: "slack#/functions/canvas_update_content",
+  source_file: "",
+  title: "Update a canvas",
+  input_parameters: {
+    properties: {
+      canvas_update_type: {
+        type: SchemaTypes.string,
+        description: "Type of update",
+        title: "Type of update",
+      },
+      channel_id: {
```

Comment:
Yes in this case the field need to be populated based on the `canvas_update_type`

If `canvas_update_type` is set to `standalone` then a `canvas_id` need to be provided, otherwise if is set to `channel_canvas` a `channel_id` need to be provided

Reply:
I see 🤔 and there is not real way to communicate this behavior through the function definition so we can't adapt the generation script to implement this in the SDK, implementing this seems like a big lift, until the community requests this feature I think relying on documentation will be the solution

Reply:
Agree we should document this behaviour